lxd/certificates.go

```diff
@@ -66,21 +68,61 @@ type certificatesPostBody struct {
 	Password    string `json:"password"`
 }
 
+func readSavedClientCAList(d *Daemon) {
+	certdbname := shared.VarPath("lxd.db")
+	db, err := sql.Open("sqlite3", certdbname)
+	if err != nil {
+		fmt.Printf("Error reading certificates from database: %s\n", err)
+		return
+	}
+	defer db.Close()
+
+	d.clientCerts = make(map[string]x509.Certificate)
+	rows, err := db.Query("select fingerprint, type, name, certificate from certificates")
```

Comment:
Similar to the create script, I'd prefer to have SQL queries using the standard "uppercase for query keywords, lowercase for DB, table and field names", so in this case:

    "SELECT fingerprint, type, name, certificate FROM certificates"